src/Routes.js

```diff
@@ -23,12 +24,20 @@ import Programs from "./organizations/Programs";
 import About from "./home/About";
 import Benefit from "./home/Benefit"
 import Nav from "./Navigation"
+import Sidebar from "./sidebar/Sidebar"
 
 
 export default function Routes() {
+  const { user } = useContext(AuthContext);
```

Comment:
@arkumari2000 , instead of doing this, I suggest you use the `isAuth` data from `AuthContext` from inside the Sidebar component as it was used to toggle between the `Register/Login` and `Logout` (line 91-116 Navigation.jsx).

<img width="583" alt="Screen Shot 2021-06-18 at 11 27 39 am" src="https://user-images.githubusercontent.com/29667122/122492252-c4906a00-d028-11eb-9893-b00a06d3dd09.png">

<img width="1027" alt="Screen Shot 2021-06-18 at 11 32 16 am" src="https://user-images.githubusercontent.com/29667122/122492307-deca4800-d028-11eb-9dbf-5e68a6073a6f.png">